proto/spire-next/api/server/svid/v1/svid.proto

```diff
@@ -111,13 +111,13 @@ message NewJWTSVIDResponse {
 }
 
 message NewDownstreamX509CARequest {
-    // Required. The entry ID for the identity being requested.
-    string entry_id = 1;
-
     // Required. The ASN.1 DER encoded Certificate Signing Request (CSR). The
     // CSR is only used to convey the public key; other fields in the CSR are
     // ignored. The X509-SVID attributes are determined by the entry.
-    bytes csr = 2;
+    bytes csr = 1;
+
+    // No entry_id there are a small number of downstream IDs and we can just 
+    // look for one that matches the CSR
```

Comment:
This comment feels more like an internal detail and is also incorrect (we don't match up anything with the CSR). I'd suggest removing it.